llpc/test/shaderdb/core/ObjNonUniform_TestTexutreLoadStoreInt64.spvasm

```diff
@@ -0,0 +1,180 @@
+               OpCapability Shader
+               OpCapability Int64
+               OpCapability ShaderNonUniform
+               OpCapability RuntimeDescriptorArray
+               OpCapability SampledImageArrayNonUniformIndexing
+               OpExtension "SPV_EXT_descriptor_indexing"
+          %1 = OpExtInstImport "GLSL.std.450"
+               OpMemoryModel Logical GLSL450
+               OpEntryPoint Fragment %main "main" %_4 %_3
+               OpExecutionMode %main OriginUpperLeft
+               OpSource GLSL 450
+               OpSourceExtension "GL_ARB_gpu_shader_int64"
+               OpSourceExtension "GL_EXT_nonuniform_qualifier"
+               OpName %main "main"
+               OpName %_12 "_12"
+               OpName %_4 "_4"
+               OpName %_3 "_3"
+               OpName %_11 "_11"
+               OpDecorate %_4 Flat
+               OpDecorate %_4 Location 1
+               OpDecorate %_3 Location 0
+               OpDecorate %_11 DescriptorSet 0
+               OpDecorate %_11 Binding 7
+               OpDecorate %26 NonUniform
+               OpDecorate %28 NonUniform
+               OpDecorate %29 NonUniform
+       %void = OpTypeVoid
+          %3 = OpTypeFunction %void
+       %uint = OpTypeInt 64 0
+%_ptr_Function_uint = OpTypePointer Function %uint
+     %v2uint = OpTypeVector %uint 2
+%_ptr_Input_v2uint = OpTypePointer Input %v2uint
+         %_4 = OpVariable %_ptr_Input_v2uint Input
+     %uint_0 = OpConstant %uint 0
+%_ptr_Input_uint = OpTypePointer Input %uint
+      %float = OpTypeFloat 32
+    %v4float = OpTypeVector %float 4
+%_ptr_Output_v4float = OpTypePointer Output %v4float
+         %_3 = OpVariable %_ptr_Output_v4float Output
+         %20 = OpTypeImage %float 2D 0 0 0 1 Unknown
+         %21 = OpTypeSampledImage %20
+%_runtimearr_21 = OpTypeRuntimeArray %21
+%_ptr_UniformConstant__runtimearr_21 = OpTypePointer UniformConstant %_runtimearr_21
+%_ptr_Function_type_2d_image = OpTypePointer Function %21
+        %_11 = OpVariable %_ptr_UniformConstant__runtimearr_21 UniformConstant
+%_ptr_UniformConstant_21 = OpTypePointer UniformConstant %21
+    %v2float = OpTypeVector %float 2
+    %float_0 = OpConstant %float 0
+         %32 = OpConstantComposite %v2float %float_0 %float_0
+       %main = OpFunction %void None %3
+          %5 = OpLabel
+        %_12 = OpVariable %_ptr_Function_uint Function
+        %texture = OpVariable %_ptr_Function_type_2d_image Function
+         %14 = OpAccessChain %_ptr_Input_uint %_4 %uint_0
+         %15 = OpLoad %uint %14
+               OpStore %_12 %15
+         %25 = OpLoad %uint %_12
+         %26 = OpCopyObject %uint %25
+         %28 = OpAccessChain %_ptr_UniformConstant_21 %_11 %26
+         %29 = OpLoad %21 %28
+               OpStore %texture %29
+  %loadImage = OpLoad %21 %texture
+         %33 = OpImageSampleImplicitLod %v4float %loadImage %32
+               OpStore %_3 %33
+               OpReturn
+               OpFunctionEnd
+; CHECK-LABEL: @main(
```

Comment:
The CHECK- lines can be removed, right? I only see the one RUN line that uses SHADERTEST